ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,154 @@
+# ML Survey Service Config
+# Application host
+HOST={{ml_survey_host | default("localhost")}}
+
+# Application port number which the service will listen to
+PORT={{ml_survey_port | default("7201")}} 
+
+# Loging type
+LOG={{ml_survey_log | default("debug")}}
```

Comment:
what are the options we can pass? by default we need warn level logs. Please check and update.

Reply:
they have removed this one from code so i will remove this